### ISSUE TITLE
Integration fails to load

### ISSUE THREAD

**Issue by amitfin** [GuyKh/ims-custom-component#116]
IMS: 0.1.29
HA: 2024.10.2

```
2024-10-13 09:19:49.904 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved (None)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 2219, in async_forward_entry_setup
    result = await self._async_forward_entry_setup(entry, domain, True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 2234, in _async_forward_entry_setup
    if domain not in self.hass.config.components:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: unhashable type: 'list'
```

Tried to reinstall with the same result.

**Comment by GuyKh** [comment]
@amitfin should be fixed with 0.1.30

**Comment by GuyKh** [comment]
Related to #114
